source/server/configuration_impl.cc

```diff
@@ -138,6 +145,16 @@ void MainImpl::initializeStatsSinks(const envoy::config::bootstrap::v3::Bootstra
   }
 }
 
+void MainImpl::initializeGRPCStreamDemuxers() {
+  ENVOY_LOG(info, "loading gRPC stream demuxer configurations");
+
+  auto& factory = Config::Utility::getAndCheckFactoryByName<Demuxer::GRPCStreamDemuxerFactory>("grpc_stream_demuxer");
+  Demuxer::GRPCStreamDemuxerPtr demuxer = factory.createGPRCStreamDemuxer();
+  // TODO (vmunukutla): It might be too early to start the demuxer here. Check if demuxer should
```

Comment:
I would actually include a blank line in this case (see 3rd bullet in https://google.github.io/styleguide/cppguide.html#Vertical_Whitespace):
```suggestion
  Demuxer::GRPCStreamDemuxerPtr demuxer = factory.createGPRCStreamDemuxer();
  
  // TODO (vmunukutla): It might be too early to start the demuxer here. Check if demuxer should
```